### PR TITLE
Update user guide

### LINKED ISSUE #125: User Guide
Pending Tasks:
- [x] Add examples for all features
- [x] Tidy up command summary with format
- [x] Change format from header to bold (so it doesn't appear in table of contents)
- [x] Add icons for better readability e.g. information icon
- [x] Add information for valid/invalid fields

### LINKED ISSUE #141: [PE-D][Tester A] Difference between UG UI and actual UI
Above is the actual UI from the .jar file. We start at the dashboard page.

Whereas above is the UI from the UG, which is supposed to be the "GUI you see on start-up".

I suspect this is because the left portion of the main window was a recent addition, and the UG has not been updated yet. 

Not a big issue, just remember to regularly update the UG! :)

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: Elijah5399/ped#1

### LINKED ISSUE #142: [PE-D][Tester B] Start Image shown in PE is diffirent from the actual one.
This is the start image shown in the UG.

This is the actual image appeard when open the jar file.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: Heran9/ped#1

### LINKED ISSUE #150: [PE-D][Tester C] Different start up screen compared to the one shown in the UserGuide
In the UserGuide, it states that the start up screen after starting the application is a list of clients as shown in the screenshot below.

However, the start up screen when I run the application is the dashboard as shown in the screenshot below.

Perhaps can switch the start up screen to show the correct start up screen displayed when a user starts the application?

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: Jweng88/ped#2

### LINKED ISSUE #155: [PE-D][Tester D] Duplicate phone numbers and emails are allowed to be added
Steps to reproduce:
1. `create n/alex yeoh p/87438807 e/alexyeoh@example.com a/123 qwert rd` create a new client
2. `create n/Alex yeoh p/87438807 e/alexyeoh@example.com a/123 qwert rd` create a new client with all fields identical except for their name

Expected: Client should not be able to be created as they have the same phone and email address, fields that are likely to be unique to only 1 individual

Actual: Client is able to be created

Given the new client has the exact same phone number and email address as the previous client, there may be a high likelihood that this is the same client 💼 In this case, the creation should not be allowed, as it is the same person behind the contact.

A quick suggestion from me is to do your duplicate checking using the phone and email fields as well, as these should be unique to each individual and will prevent users from adding duplicate clients!

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: ivanleekk/ped#2

### LINKED ISSUE #160: [PE-D][Tester B] Ambiguous description of method
Actually I thought we can simply use the find command to search incomplete words such as find "Ale" will return Alex Yeoh, but it seems actually we can only provide first name or last name.

Personlly speaking, better to clarify it in the UG.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: Heran9/ped#4

### LINKED ISSUE #161: [PE-D][Tester D] Clients with the identical name cannot be added
Steps to reproduce:
1. `create n/Alex Alan p/12345678 e/a@test.com a/123 qwert rd` to create a new client Alex Alan
2. `create n/Alex Alan p/87654321 e/a@example.com a/321 asdf rd` to create a new client who has a different phone and email and address, but same name

Expected: 2nd Client is able to be created, as a name may be very common, and since phone and email are different it is likely to be a different client

Actual: Client could not be added as an error message was thrown saying that the person already exists in the address book

As some name may be very common, a repetition is an inevitability, especially when dealing with clients, you may only get their first or last name. At such instances if a client has the same name as a previous client, as a user I will be unable to add him/her to the client list and save their details.

A quick suggestion from me is to have the duplicate checking ignore the name field and focus on the phone and/or email fields as those should be unique to each individual, allowing you to prevent duplicates whilst still allowing for people with the same name!

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: ivanleekk/ped#3

### LINKED ISSUE #176: [PE-D][Tester C] Unable to delete created tag for a client profile
In the UserGuide, there were only mentions of deleting a client, but no mentions of deleting created tags. In the event of a user accidentally creating tags or lead, the user is unable to undo the action as the only way currently is to manually edit the `addressbook.json` file. Perhaps can consider providing users a way to remove unwanted tags or lead from a client profile?

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: Jweng88/ped#11

### PR DESCRIPTION
**Pull request by dickongwd**
Fixes #125.

### Changes
* Updated images: fixes #150, fixes #141, fixes #142.
* Added section about unique profiles: fixes #155, fixes #161.
* Added section about how to clear tags: fixes #176.
* Clarified **find** command: fixes #160.
* Updated examples for all commands (except delete).

### CODE REVIEW

**Review by lilozz2 — APPROVED** [review]
LGTM

**Review by MagnificentCreature — APPROVED** [review]
LGTM

**Comment by zhyuhan** [comment]
We should add that a client is uniquely identified by their name (see #155, #161).

File: docs/UserGuide.md [hunk 1]
```diff
@@ -182,33 +257,55 @@ You might want to go back to the list of clients you have added to Connectify. U
...
-find John Doe
-```
+**:information_source: Notes about the find command**<br>
 
-Example:
-Finding a client with the name "Chemmy Lee". Notice that you don't need to type the full name of the client.
-```
-find Chemmy
+You need to type either the **first** name or the **last** name of the client. You cannot type part of the client's name!
+
```
Comment (KamiliArsyad) [inline docs/UserGuide.md]: I think just now I reviewed a PR that also modified this but I'm not sure which one so j be careful later when merging either coz I'm p sure both are alr approved

**Review by KamiliArsyad — APPROVED** [review]
alls good lgtm

**Pull request merged**